lib/UserBackend.php

```diff
@@ -627,26 +627,34 @@ public function updateAttributes($uid,
 		$user = $this->userManager->get($uid);
 		try {
 			$newEmail = $this->getAttributeValue('saml-attribute-mapping-email_mapping', $attributes);
+			$this->logger->debug('Email attribute content: '. $newEmail);
```

Comment:
```suggestion
			$this->logger->debug('Email attribute content: {email}', ['app' => 'user_saml', 'email' => $newEmail]);
```

and correspondingly to the following statments